### ISSUE TITLE
Community Discovery + Public Interaction

### ISSUE THREAD

**Issue by 0chroma** [0chroma/recipra#6]
For more open communities I think it's good to start thinking about how new members could discover them. Some notes I have for potential solutions:

- public facing group content? could lead to superficial interaction, lame
- hosted events open to all?
  - meet and greets
  - public workshops
  - gatherings hosted by the group
  - streams w/ call-ins or fan submissions
  - twitter-spaces like function?
- hashtag based discovery w/ member interviews/tryouts/provisional membership?
- volunteer work ie pull requests?

**Comment by riiadesign** [comment]
thats a great direction to go in

i think the public facing group content can be viewed but a sign in would be necessary to interact

that way you can have a cta to sign up if people like what they see

alternatively, instagram's route is a truncated post forcing the viewer to download/sign up to view the post - i believe this is a dark ux route though, and dont recommend it

**Comment by 0chroma** [comment]
yeah that's true, public content would allow people to share things and could help the app catch on. Though, if we're going for a replacement for Discord, maybe public content isn't needed? Not sure.

**Comment by 0chroma** [comment]
An idea I came up with today would be to publish things through something like IPFS, which has gateways to regular HTTP. Using Hyphanet could work too, but I'm not aware of any gateways for this.

That way you could, as an example, link someone outside the group to upload photos to a photoboard, etc. Would still need some type of anonymity for this, see https://github.com/0chroma/InductionDB/issues/1
